python/device/device.py

```diff
@@ -177,147 +169,143 @@ def _configure_azure_monitor_metrics(self):
         # System Health metrics
         # ---------------------
         self.reporter.add_float_measurement(
-            MetricNames.PROCESS_CPU_PERCENT, "Amount of CPU usage by the process", "percentage",
+            Metrics.PROCESS_CPU_PERCENT,
+            "CPU use for the device app, as a percentage of all cores",
+            "percentage",
         )
         self.reporter.add_integer_measurement(
-            MetricNames.PROCESS_WORKING_SET, "All physical memory used by the process", "bytes",
+            Metrics.PROCESS_WORKING_SET,
+            "Working set for the device app, includes shared and private, read-only and writeable memory",
+            "bytes",
         )
         self.reporter.add_integer_measurement(
-            MetricNames.PROCESS_BYTES_IN_ALL_HEAPS,
-            "All virtual memory used by the process",
+            Metrics.PROCESS_BYTES_IN_ALL_HEAPS,
+            "Size of all heaps for the device app, essentially 'all available memory'",
             "bytes",
         )
         self.reporter.add_integer_measurement(
-            MetricNames.PROCESS_PRIVATE_BYTES,
-            "Amount of non-shared physical memory used by the process",
-            "bytes",
+            Metrics.PROCESS_PRIVATE_BYTES, "Amount of private data used by the process", "bytes",
```

Comment:
What's the difference between the `PROCESS_PRIVATE_BYTES` and `PROCESS_WORKING_SET_PRIVATE` metrics?

Reply:
I think there is no difference, but .net has 2 different measurements for these.  I haven't dug as deeply as I would like into this -- the 2 will probably be collapsed into 1 here at some point.